components/formats-gpl/src/loci/formats/in/MetamorphReader.java

```diff
@@ -352,7 +351,7 @@ protected void initFile(String id) throws FormatException, IOException {
       LOGGER.info("Initializing " + id);
       // find an associated STK file
       String stkFile = id.substring(0, id.lastIndexOf("."));
-      if (stkFile.indexOf(File.separator) != -1) {
+      if (stkFile.indexOf(File.separatorChar) != -1) {
```

Comment:
http://docs.oracle.com/javase/7/docs/api/java/io/File.html#separatorChar and http://docs.oracle.com/javase/7/docs/api/java/lang/String.html#indexOf%28int%29 for convenient double-checking. :smiley: